fabric_cf/orchestrator/core/orchestrator_slice_wrapper.py

```diff
@@ -127,11 +127,27 @@ def create(self, *, bqm_graph: Neo4jCBMGraph, slice_graph: Neo4jASM,
             slivers = []
             for nn_id in slice_graph.get_all_network_nodes():
                 sliver = slice_graph.build_deep_node_sliver(node_id=nn_id)
+
+                # Compute Requested Capacities from Capacity Hints
+                requested_capacities = sliver.get_capacities()
+                requested_capacity_hints = sliver.get_capacity_hints()
+                catalog = InstanceCatalog()
+                if requested_capacities is None and requested_capacity_hints is not None:
+                    requested_capacities = catalog.get_instance_capacities(
+                        instance_type=requested_capacity_hints.instance_type)
+                    sliver.set_capacities(cap=requested_capacities)
+
+                # Compute Capacity Hints from Requested Capacities
+                if requested_capacity_hints is None and requested_capacities is not None:
```

Comment:
What happens if both capacities and capacity_hints are none?

Reply:
CF code doesn't check it explicitly but I can add those validations. Request would end up failing at the Broker when being Ticketed. It would be more efficient to check at orchestrator.